src/jquery/core.js

```diff
@@ -106,6 +106,10 @@ if ( jQueryVersionSince( "3.2.0" ) ) {
 		return elem.nodeName && elem.nodeName.toLowerCase() === name.toLowerCase();
 	},
 	"jQuery.nodeName is deprecated" );
+
+        migrateWarnFunc( jQuery, "isArray", Array.isArray,
+                "jQuery.isArray is deprecated; use Array.isArray"
+        );
```

Comment:
Please follow our code style which uses tabs, not spaces.

Reply:
done